packages/quick_actions/quick_actions_ios/ios/Classes/MethodChannel.swift

```diff
@@ -0,0 +1,16 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Flutter
+
+/// A channel for platform code to communicate with the Dart code.
+protocol MethodChannel {
+  /// Invokes a method in Dart code.
+  /// - Parameter method the method name.
+  /// - Parameter arguments the method arguments.
+  func invokeMethod(_ method: String, arguments: Any?)
+}
+
+/// A default implementation of the `MethodChannel` protocol.
+extension FlutterMethodChannel: MethodChannel {}
```

Comment:
Here I put this extension in the same file as the protocol. More details discussed in [Rule 3(b) of the Swift POP naming convention proposal](https://docs.google.com/document/d/1H8F-0OmJ5GF4BCwfC_j7rs7k02dLkrcxpe2IZbaQg5Q/edit#heading=h.nx4eex3rae73).